unity/Assets/Maroon/reusableScripts/Physics/Electromagnetism/IField.cs

```diff
@@ -83,4 +83,8 @@ public abstract class IField : MonoBehaviour
     public abstract float getStrength(Vector3 position);
 
     public abstract float getStrengthInPercent(Vector3 position);
+
+    public abstract void addProducerToSet(GameObject producer);
+
+    public abstract void removeProducerFromSet(GameObject producer);
```

Comment:
I know other methods in this class also do not follow that, but method names should start with capital letters, 
IMO we should still try to name new methods correctly, even if old ones are not named correctly, maybe we can fix those old names sometime